### DISCUSSION TITLE
Sky Q e Spotify

### DISCUSSION THREAD

**Discussion by mriitan** [indomus/forum#1419] [HUB: Home Assistant]
ciao a tutti,
ho integrato skyq e spotify, ma volevo sapere se per sky era possibile vedere anche il contenuto e non solo vedere che sia acceso e leggere cosa si sta guardando, insomma vedere su tablet o pc il contenuto al momento in produzione.
Invece per spotify ho visti che si deve mettere in esecuzione su un device, non è invece possibile mandarlo in esecuzione direttamente dal tablet e dalla schermata HA?
grazie

**Comment by snakuzzo** [comment]
Per SkyQ -> Il custom component lo fa di suo -> https://indomus.it/guide/integrare-skyq-alla-domotica-home-assistant/ 
Per Spotify -> devi inviare la playlist ad un'entità di tipo media player
